lib/shortcuts/queue-lambda.js

```diff
@@ -2,18 +2,20 @@
 
 const Lambda = require('./lambda');
 
+/**
+ * @typedef { import('./lambda').LambdaOptions } LambdaOptions
```

Comment:
Neat - we can import the base `Lambda` class's `typedef` for its constructor `options` arg.